File: js_solutions/0009_palindrome_number.js
/**
 * @link https://leetcode.com/problems/palindrome-number/
 * @description Check if given number is a palindrome
 * @param {Number} number
 * @returns {Boolean}
 */
const isPalindrome = (number) => {
  const string = number.toString();
  for (let i = 0, j = string.length - 1; i < j; i += 1, j -= 1) {
    if (string[i] !== string[j]) {
      return false;
    }
  }
  return true;
};

File: python_solutions/0009_palindrome_number.py
def isPalindrome(self, x: int) -> bool:
  string = str(x)
  i = 0
  j = len(string) - 1
        
  while (i < j):
    if (string[i] != string[j]):
      return False
    i += 1
    j -= 1
            
  return True